database/sql/V072__auction_solution_orders.sql

```diff
@@ -0,0 +1,66 @@
+-- All auctions ran by autopilot
+CREATE TABLE competition_auctions (
```

Comment:
Let's try to avoid these composite table names if possible as I think they mostly cause confusion.
I'd say this should be called `auctions` and the current auctions table would become `current_auction` (singular as it's supposed to only store a single row at all times).

Reply:
I'd like to avoid touching existing code with this PR. Table renaming is particularly risky, and even though I initially wanted to do renaming, I went with defining a new name after all. 
And if you assume that, it's really hard to figure out a new name for this table.

Reply:
Then at least make sure the tables are properly renamed when we finalize this refactor and remove the old tables.